catalog/game.rb

```diff
@@ -52,9 +51,10 @@ def self.save_all(games)
     File.write(file_path, JSON.pretty_generate(data))
   end
 
-  private
+  # private
 
-  def age_in_years
-    Date.today.year - Date.parse(@last_played_at).year
-  end
+  # def age_in_years
+  #   today = Date.today
+  #   Date.new(today.year - 2, today.month, today.day)
+  # end
```

Comment:
- [ optional ] Great job so far. It's always good to follow best practices. Kindly keep your code clean by removing the commented code.